main/lsp/LSPTypechecker.cc

```diff
@@ -527,20 +517,22 @@ LSPTypechecker::SlowPathResult LSPTypechecker::runSlowPath(LSPFileUpdates &updat
             return;
         }
 
-        if (this->config->opts.stripePackages) {
-            // Only need to compute FoundDefHashes when running to compute a FileHash
-            auto foundHashes = nullptr;
-            auto cancelled =
-                pipeline::name(*this->gs, absl::MakeSpan(indexed), this->config->opts, workers, foundHashes);
-            if (cancelled) {
-                ast::ParsedFilesOrCancelled::cancel(move(indexed), workers);
-                ast::ParsedFilesOrCancelled::cancel(move(nonPackagedIndexed), workers);
-                return;
-            }
-        }
+        // First run: only the __package.rb files. This populates the packageDB
+        pipeline::buildPackageDB(*this->gs, absl::MakeSpan(indexed), this->config->opts, workers);
```

Comment:
Should this be guarded by a check for stripe-packages being enabled?

Reply:
It doesn't need to be. I wrote about this in the commit message:

https://github.com/sorbet/sorbet/pull/8745/commits/aa67c0f1518de175b2a9136f94fd1b48dd13e71d

but the tl;dr is that if `stripePackages` is `false`, then `indexed` will be empty and thus this call is a no-op. That's how it works in realmain as well.